src/components/react/DrawerLinks.tsx

```
@@ -0,0 +1,100 @@
+  linkConfigs,
+  className,
+}: {
+  linkConfigs: LinkConfigs[];
+  className: string;
+}) {
+  const cls = cn(className, 'pb-6 grid gap-2');
+  return (
+    <ul className={cls}>
+      {linkConfigs.map(({ href, icon: Icon, text }) => (
+        <li>
+          <a
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)